app/Http/Controllers/Api/UserController.php

```diff
@@ -43,7 +42,7 @@ public function repositories() : \Illuminate\Support\Collection
             ->selectName()
             ->selectCreatedAt()
             ->selectNamespace()->selectName()->selectFullName();
-        $token = App::environment(['local']) ? env('GITLAB_ACCESS_TOKEN') : auth()->user()->access_token;
```

Comment:
why we do not use user access token?

Reply:
We don't want it since there is no refresh flow in place.

So if you are logged in for more than 1 hour, the access_token generated on login, will expire making the search fail.

This also leads to another problem, since the "Bot account" is the one that creates and forks projects, but the search was based on your own, you could find more repositories than the bot had access to making it fail to create the assignment, since it does not have access to fork a repo it is not a member of.